framework/util/platform.h

```diff
@@ -288,28 +293,49 @@ inline void* GetProcAddress(LibraryHandle handle, const char* name)
     return dlsym(handle, name);
 }
 
+#if defined(__ANDROID__)
+static const prop_info* GetAndroidPropertyInfo(const char* name)
+{
+    // According to system_properties.h "Property lookup is expensive, so it can be useful to cache the result of this
+    // function.". For that reason we cache property infos.
```

Comment:
I saw that comment too, how much overhead is it? Was not sure it is worth the added complexity.